linux/update_vcpkg_manager.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/sh
+set -e
+
+if [ ! -d "linux" ]; then
+    echo "start this script in the source root directory"
+    exit 1
+fi
+
+CACHE_DIR="$PWD/3rdParty/buildCache/linux"
+BUILD_DIR="$PWD/3rdParty/linux"
+VCPKG_PORTS="$PWD/3rdParty/vcpkg_ports"
+VCPKG_ROOT="$BUILD_DIR/vcpkg"
+
+if [ ! -d $VCPKG_ROOT ]; then
+    mkdir -p $BUILD_DIR
+    git -C $BUILD_DIR clone https://github.com/microsoft/vcpkg
+fi
+
+git -C $VCPKG_ROOT pull
+$VCPKG_ROOT/bootstrap-vcpkg.sh
+$VCPKG_ROOT/vcpkg install curl[core,openssl] wxwidgets --clean-after-build --overlay-triplets=$VCPKG_PORTS/triplets/ci
```

Comment:
I assume this should use now manifest mode

Reply:
We cannot do manifest mode. wxwidgets need to be installed separate because of it will not install with free type and other depends of fglut.

Reply:
I don't understand. Could you please rephrase it?

Reply:
Please ask on the part you don't understand.

Reply:
Please be more specific that I can answer you.

Reply:
Why can't you install wxwidgets and all its dependencies using vcpkg manifest?

Reply:
Because `wxwidgets` port will not install together with `freetype`  It need install first wxwidgets then another command to install the rest. it cannot done in manifest mode.
You can read more here: 
https://github.com/microsoft/vcpkg/pull/24047#issuecomment-1098377279

Reply:
Is there an open ticket for this issue?

Reply:
It not an issue.

Reply:
If some part of functionality doesn't work as designed - it's an issue

Reply:
ok. You can open an issue on vcpkg, and tell them that.